src/constants/department_maps/major-style.map.ts

```
@@ -26,6 +26,7 @@ export const MAJOR_STYLE_MAP: Record<string, string> = {
     ARTSPORTS: '예술체육대학',
     SWCC: '소프트웨어융합대학',
     GENERALEDU: '프런티어창의대학',
+    BIOSYST: '바이오시스융합학부',
```

ESLint

[error] Delete `··`

(prettier/prettier)